docker/Dockerfile.cross-in-docker

```diff
@@ -0,0 +1,10 @@
+FROM rust:latest
+
+ENV CROSS_DOCKER_IN_DOCKER=true
+
+ARG checksum="d9d9b964d428733e98c2579b91772af4  /usr/bin/docker"
+
+RUN wget -O- https://download.docker.com/linux/static/stable/x86_64/docker-20.10.9.tgz | tar -C /usr/bin/ --strip-components=1 -xzf - docker/docker
+RUN echo $checksum | md5sum -c
+
+RUN cargo install --git https://github.com/cross-rs/cross
```

Comment:
This is still wrong, it should copy from the checked out code, if this is kept the CI will work, but it'll check the most recent release, not the proposed changes

Reply:
Sure, it should use the checkout for building, thanks @Emilgardis for raising this up.